app/vmctl/vm_native_test.go

```diff
@@ -110,7 +110,7 @@ func Test_vmNativeProcessor_run(t *testing.T) {
 
 			tt.closer(cancelFn)
 
-			if err := p.run(ctx); (err != nil) != tt.wantErr {
+			if err := p.run(ctx, true); (err != nil) != tt.wantErr {
```

Comment:
can we make this test runnable without preconditions? I'm afraid it is impossible to tell if any change affects the test because they don't actually run in CI.

Reply:
If you do not mind i will better tests in the next PR